oddbird/odddocker
# Add keep-alive commit to build job

![](https://source.unsplash.com/featured/?cute,animal&220204)



## Description
Adds a script at the end of the build job that will commit to the `gh-actions-keep-alive` branch if no commits have happened in `main` for more that 50 days. This should keep us out of the [60 day restriction](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#disabling-and-enabling-workflows).


## Steps to test/reproduce
I guess we'll have to wait for 50 days without committing to the repo to see if this works 🤷‍♂️. The script has been tested locally and the commit will be made, I'm just not sure if GitHub will lift the restriction if we don't push to `main`.
